core/src/scene/scene.h

```diff
@@ -31,6 +31,8 @@ struct Stops;
  * Scene is a singleton containing the styles, lighting, and interactions defining a map scene
  */
 
+using GlobalUpdate = std::pair<std::string, YAML::Node>;
```

Comment:
Abstraction is cool, but this type definition is used in exactly one place so in this case we're not clarifying anything - let's remove it.